tee-worker/ts-tests/common/transactions.ts

```diff
@@ -158,7 +158,6 @@ export async function listenEvent(api: ApiPromise, section: string, methods: str
                     }
                     return (
                         phase.isApplyExtrinsic &&
-                        phase.asApplyExtrinsic.eq(index) &&
```

Comment:
Don't we need this to make sure that each event gets mapped to the correct extrinsic? :thinking:

Reply:
I made a mistake, the CI error had nothing to do with this change, so I added it back. :(